gateway/1.8/modules/ROOT/pages/flex-architecture-multi-region.adoc

```diff
@@ -74,14 +74,14 @@ In both <<implementation-3a>> and <<implementation-3b>>, Flex Gateway directs tr
 
 This implementation is similar to <<implementation-1a>> and has the same considerations. However, Redis replication is not required because requests are sent to specific regions. Configure high availability or disaster recovery zones in each region to ensure API availability:
 
-image:Flex-Gateway-DR-4.png["A detailed view of implementation 3A, which contains the necessary services to enable route customer requests to the closest region."]
+image:high-availability-3a.png["A detailed view of implementation 3A, which contains the necessary services to enable route customer requests to the closest region."]
```

Comment:
```suggestion
image:high-availability-3a.png["A detailed view of implementation 3A, which contains the necessary services to enable routing customer requests to the closest region."]
```

Reply:
An alternative might be something like "enable Flex Gateway to route ..." or just "the necessary services to route customer ..."